dc/council/code/titles/26/sections/26-1113.xml

```diff
@@ -50,31 +50,31 @@
-        <text>“(C) Lender ..........</text>
-        <text>“(D) Lender Address ..........</text>
-        <text>“(E) Lender Phone Number ..........</text>
-        <text>“(F) Your loan is for $ ..........       , for a term of ..........       years. The final maturity date is ..........</text>
-        <text>“Your beginning interest rate is ___       %. This rate is good for ___       months/years [circle one]. This rate and your payment can increase, starting on [date]       , and may continue to increase, depending on the terms of your mortgage.</text>
-        <text>“(G) Beginning on ..........       , you will be charged at the fully-indexed rate, which is your margin (..........       %) plus an index value, which for you is ..........       . Estimating based on the current rate of the index, which is ..........       %, your monthly payment at the fully-indexed rate would be $ ..........       . While the index rate does vary, your mortgage provides that the fully-indexed rate will not rise above ..........       %. At that rate, your monthly payment would be $..........</text>
-        <text>“(H) YOU HAVE INDICATED THAT YOUR GROSS MONTHLY INCOME IS $ ..........</text>
-        <text>“(I) WARNING: Industry standards suggest that a homeowner should spend no more than 28% of his or her gross monthly income on mortgage costs (including taxes and insurance).</text>
-        <text>“(J) $ ..........       /month = Your principal + initial interest + taxes and insurance.</text>
-        <text>“(K) $ ..........       /month = Your principal + adjusted interest + taxes and insurance.</text>
-        <text>“(L) $ ..........       /month = Your principal + maximum interest + taxes and insurance.</text>
-        <text>“(M) $ ..........       /month = 28% of your current gross monthly income (the recommended limit).</text>
-        <text>“(N) Your gross monthly income may rise or fall over time, but if either of the first 3 amounts exceeds the fourth, you may want to reconsider the suitability of this loan for your needs. You may cancel your mortgage application within 5 business days of receiving this form.</text>
-        <text>“(O) Your mortgage carries a balloon payment. This means that on ......       , you will have to fully pay the remaining balance on the loan.</text>
-        <text>“(P) Your loan has a prepayment penalty. This means that if you pay off your mortgage in the first ..........       years, you will have to pay a penalty of $ ..........       . If you refinance your mortgage in that period, you will be required to pay this amount.</text>
-        <text>“(Q) See definitions of underlined terms on reverse side. DO NOT SIGN THIS IF YOU DO NOT UNDERSTAND IT!</text>
-        <text>“....................</text>
-        <text>“Lender’s Authorized Representative and date</text>
-        <text>“....................</text>
-        <text>“Borrower(s) and date.”</text>
-      </para>
+      <text>
```

Comment:
Point taken - I see where you are coming from. I think, perhaps, we are both correct? 

It is a form agreement, as you state, but it is using for that form the "THE INTERSTATE COMPACT FOR JUVENILES", which is a document in its own right. (Here is a link I quickly found that seems to match this on cursory review: [THE INTERSTATE COMPACT FOR JUVENILES](http://www.juvenilecompact.org/Portals/5/library/legal/ICJRevisedLanguage.pdf)).

Thus, the form is including that document in its entirety.

Which is the correct way, to go? I propose we err on the side of keeping the structure. But, I will leave that to you and @dgreisen to make the final call.